liberty-archetype-ear/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.openliberty.tools</groupId>
         <artifactId>liberty-maven</artifactId>
-        <version>3.0.2-SNAPSHOT</version>
+        <version>3.0.1</version>
```

Comment:
I think there was some confusion here. All the archetypes should still have the latest version of the parent. The plugin version that needs to be the latest preceding release is actually in each archetype's `archetype.properties` file. For example, this file for the `liberty-archetype-ear` is here: https://github.com/OpenLiberty/ci.maven/blob/master/liberty-archetype-ear/src/test/resources/projects/it/archetype.properties

When an archetype is generated, the defaults will be derived based on the values from this file. We use these defaults to generate test projects for each archetype. We do not want a snapshot version to be the default, which is why I said that the plugin version should be set to the latest preceding release in the archetype properties.

So all these archetype poms should still define 3.1-SNAPSHOT as the parent, and the `archetype.properties` should define the plugin version as the latest preceding release.

Reply:
Yes, you said to change the archetypes to 3.0.1...so that is what I did I thought.